StudyHub/StudyHub/Extensions/ConfigExtension.cs

```diff
@@ -0,0 +1,12 @@
+﻿using StudyHub.Common.Configs;
+
+namespace StudyHub.Extensions;
+
+public static class ConfigExtension
+{
+    public static IServiceCollection AddConfig<T>(this IServiceCollection services, IConfiguration configuration)
+        where T : ConfigBase
+    {
+        return services.Configure<T>(configuration.GetSection(typeof(T).Name));
```

Comment:
this adds config to DI as IOptions\<T\>, maybe we can directly add Config instance